test/e2e/pd.go

```diff
@@ -134,6 +135,66 @@ var _ = framework.KubeDescribe("Pod Disks", func() {
 		return
 	})
 
+	It("Same test as above, but remove pod with default grace period 30s [Slow]", func() {
```

Comment:
Can we make this name: "Should schedule a pod w/ a RW PD, gracefully remove it, then schedule it on another host [Slow]"